documentation/modules/proc-installing-using-template.adoc

```diff
@@ -20,12 +20,12 @@ oc login -u system:admin
 +
 [options="nowrap",subs="attributes"]
 ----
-oc new-project enmasse
+oc new-project {ProductNamespace}
 ----
 
```

Comment:
Add:
"where <{ProductNamespace}> is the namespace where {ProductName} is deployed."